katello/katello-backup

```diff
@@ -63,6 +67,18 @@ def create_directories(directory)
   FileUtils.chmod_R 0770, directory
 end
 
+def confirm
+  puts "WARNING: This script will stop your services."
+  print "Are you sure(Y/N)? "
+  response = gets.chomp
+  if /[Y]/i.match(response)
+  else
+    puts "**** cancelled ****"
+    FileUtils.rm_rf @dir
+    exit(-1)
+  end
+end
```

Comment:
Nice! Updated. Also... probably should have something for the true case scenario, guess that would help haha